src/snappy/snappy.py

```diff
@@ -66,18 +62,35 @@
 
 def _masked_crc32c(data):
     # see the framing format specification
-    crc = _crc32c(data)
+    crc = crc32c.crc32c(data)
     return (((crc >> 15) | (crc << 17)) + 0xa282ead8) & 0xffffffff
 
-_compress = compress
-_uncompress = uncompress
+_compress = cramjam.snappy.compress
+_uncompress = cramjam.snappy.decompress
+
+
+class UncompressError(Exception):
+    pass
 
 
 py3k = False
```

Comment:
py2 stuff is for pypy only - maybe that should be dropped too

Reply:
I was thinking a followup PR for current Python versions (and dropping Python2). I actually started with #129